javaschool-backend/src/main/java/com/nearsoft/javaschoolbackend/config/RabbitMQConfig.java

```diff
@@ -0,0 +1,39 @@
+package com.nearsoft.javaschoolbackend.config;
+
+import org.springframework.amqp.core.*;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class RabbitMQConfig {
+
+    @Value("${rabbitmq.queue}")
```

Comment:
using `@Value` like this can quickly become problematic. Let's say you change the entry name in the YAML/properties file, you would break the code each place where this value is being read. Use encapsulation instead, to avoid these problems, by isolating this value the client code will see it through an abstraction (hence not know about the concrete implementation/actual entry name), it would be type safe and if you change the entry name you would only need to modify it in one place or even not at all.